src/test/scala/za/co/absa/abris/avro/parsing/utils/ExampleSchemaConverter.scala

```diff
@@ -0,0 +1,74 @@
+package za.co.absa.abris.avro.parsing.utils
+
+import scala.collection.JavaConverters._
+import scala.util.Random
+
+import org.apache.avro.{LogicalTypes, Schema, SchemaBuilder}
+import org.apache.avro.LogicalTypes.{Date, Decimal, TimestampMicros, TimestampMillis}
+import org.apache.avro.Schema.Type._
+
+import org.apache.spark.annotation.DeveloperApi
+import org.apache.spark.sql.catalyst.util.RandomUUIDGenerator
+import org.apache.spark.sql.types._
+import org.apache.spark.sql.types.Decimal.minBytesForPrecision
+
+/**
+ * This is an example of a way to build a schema that can compile with avro tools
```

Comment:
Could you add short description, how this Converter differs from the default one?